sunpy/coordinates/sun.py

```diff
@@ -73,6 +73,30 @@ def sky_position(t='now', equinox_of_date=True):
     return ra, dec
 
 
+# Time in Julian Days of the start of CR1
+_FIRST_CROT_JD = 2398167.4
+# Length of a carrington rotation in days
+_CARRINGTON_ROTATION_PERIOD = 27.2753
+
+
+def carrington_rotation_starttime(crot):
+    """
+    Return the start time of a given Carrington rotation.
+
+    Parameters
+    ----------
+    crot : int
```

Comment:
Since `carrington_rotation_number()` can return a fractional rotation number, I'm inclined to have this function also accept fractional rotation numbers so that it can do a straight inversion.